plugins/services/src/js/structs/Framework.js

```diff
@@ -51,4 +45,40 @@ module.exports = class Framework extends Application {
 
     return {value};
   }
+
+  getResources() {
+    // TODO: Circular reference workaround DCOS_OSS-783
+    const MesosStateStore = require('../../../../../src/js/stores/MesosStateStore');
```

Comment:
or we could find different approach of checking on an instance class. Currently we import Framework in the Store to check if an instance is a Framework. :man_shrugging: